test/seed/02-psychologists.ts

```diff
@@ -1,7 +1,7 @@
 import clean from '../helper/clean';
 
-import { DOSSIER_STATE } from '../../utils/dossierState';
-import { generateUuidFromString } from '../../utils/uuid';
+import { DossierState } from '../../types/DemarcheSimplifiee';
```

Comment:
We could add `types` dir in the ts config to be able to import `types/DemarcheSimplifiee` directly

Reply:
https://trello.com/c/fBF734Qm/155-create-alias-in-the-backend-just-like-in-the-frontend